README.md

```diff
@@ -1 +1,181 @@
-# planship-vue
\ No newline at end of file
+# planship-vue
+
+Welcome to `@planship/vue`, a [Vue plugin](https://vuejs.org/guide/reusability/plugins) that enables entitlements, metering, plan packaging, and customer/subscription management in your Vue app powered by [Planship](https://planship.io). This plugin is built on top of the [@planship/fetch](https://github.com/planship/planship-js/tree/master/packages/fetch) JavaScript library, and it uses the [Vue provide/inject mechanism](https://vuejs.org/guide/components/provide-inject).
```

Comment:
library, and it uses the Vue provide/inject mechanism -> library and uses Vue's [provide/inject pattern]((https://vuejs.org/guide/components/provide-inject).